src/libraries/Microsoft.Extensions.Options/src/Microsoft.Extensions.Options.csproj

```diff
@@ -1,13 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
As far as I can tell, this is only used by https://github.com/dotnet/runtime/blob/c64ebe622d187463c8aa9a38fe31ea1dc5485383/src/libraries/Microsoft.Extensions.Options/src/OptionsCache.cs which looks like it also might be satisfied by `netstandard2.1`

Reply:
The shipped configuration in the 3.1.12 package was netcoreapp3.1 and I decided to restore it instead of downgrading it further / mixing universes.

Reply:
I bet the 3.0 version had a netcoreapp3.0 asset.  We should just survey the usage and minimize configurations while maintaining compat.

Reply:
There's still the `netstandard2.0` asset which will cover netcoreapp2.x and netcoreapp3.0. As all of those platforms will be out of support the next time we will ship this package hence I'm less concerned about hose. .NET Framework has its own asset anyway. Ideally we would just drop the `netstandard2.0` asset and keep the `netcoreapp3.1` asset but I understand that this might not be possible / desirable at this point.

Reply:
I switched to netstandard2.1 to simplify the source build story here.